src/app/modules/cerf/cerf.component.ts

```diff
@@ -49,6 +49,20 @@ export const invalidRangeValidator: ValidatorFn = (control: FormGroup) : Validat
 type Attendee = {name: string, service: number, leadership: number, fellowship: number};
 // type Member = {name: string, email: string, _id: string};	// Put into Member interface to be used by MemberService too
 
+@Directive({
+	selector: '[disableControl]'
+  })
+  export class DisableControlDirective {
+  
+	@Input() set disableControl( condition : boolean ) {
+	  const action = condition ? 'disable' : 'enable';
+	  this.ngControl.control[action]();
+	}
+  
+	constructor( private ngControl : NgControl ) {
+	}
+  
+  }
```

Comment:
Would we want to use this directive in other components? Could it be extracted into it's own util directive for reusability